soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_exclude_overworld.cpp

```diff
@@ -1356,4 +1356,164 @@ void HintTable_Init_Exclude_Overworld() {
                        //obscure text
                        Text{"a #cow in a luxurious hole# offers", /*french*/"la #vache dans une grotte luxueuse# donne", /*spanish*/"una #vaca de un lujoso hoyo# brinda"},
   });
+
+    hintTable[KF_STORMS_GROTTO_BEEHIVE_LEFT] = HintText::Exclude({
```

Comment:
This hint text could be done better, even with the limits of grottos. "A beehive above X deku scrubs hides a", "A beehive in a generic grotto hides a", "A beehive above a cow hides a", "A beehive to the left/right of King Zora hides a" and "A beehive behind King Zorahides a" should cover all of them.

Reply:
I went with `a beehive above a lonely deku scrub hides`, `a beehive above a pair of deku scrubs hides`, `a beehive above a trio of deku scrubs hides`, `a beehive above a chest hides`, `a beehive above a cow hides`, `a beehive in front of the king of the zoras hides` and `a beehive behind the king of the zoras hides`.